features/dist-archive.feature

```diff
@@ -224,3 +224,37 @@ Feature: Generate a distribution archive of a project
     And the wp-content/plugins/hello-world/hello-world.php file should exist
     And the wp-content/plugins/hello-world/.travis.yml file should not exist
     And the wp-content/plugins/hello-world/bin directory should not exist
+
+Scenario: Avoids recursive symlink
+    Given a WP install in wordpress
+    And a .distignore file:
+      """
+      wp-content
+      wordpress
```

Comment:
The tests should trigger the extra regex rules as well to ensure they work as expected:
- using an asterisk
- using a slash
- using a dot